source/frontend/src/features/leases/detail/LeasePages/details/UpdateLeaseContainer.test.tsx

```diff
@@ -80,9 +81,9 @@ describe('Update lease container component', () => {
     mockAxios.onPut().reply(409, { error: 'test message' });
     await act(() => userEvent.click(getByText('Save')));
     expect(await findByText('test message')).toBeVisible();
-  });
+  });*/
 
-  it('clicking on the save anyways popup saves the form', async () => {
+  /*it('clicking on the save anyways popup saves the form', async () => {
```

Comment:
@FuriousLlama nit: please use xit instead of commenting these out, it makes skipped tests more visible.

Reply:
(or xdescribe at the top level)

Reply:
The problem is that some of these require fixes for them to compile. Given that the updated story should work on them I decided to wait. If you feel strongly about I can fix em sufficiently and disable them as you mentioned

Reply:
oh, ok. that is fine, I didn't realize there was actual compilation errors.